src/Symfony/Component/Templating/TemplateReference.php

```diff
@@ -75,7 +75,7 @@ public function all()
      */
     public function getPath()
     {
-        return $this->parameters['name'];
+        return 'views/'.$this->parameters['name'];
```

Comment:
This is most likely not the right way to solve this issue, as it forces everyone to store templates in the "views" directory. While this might be the case for Symfony projects, it's not necessarily the case for other projects using the templating component.